Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -35,30 +37,42 @@ class BrowserViewController: UIViewController {
         tabManager = TabManager(defaultNewTabRequest: defaultRequest)
     }
 
-    override func viewDidLoad() {
-        let headerView = UIVisualEffectView(effect: UIBlurEffect(style: UIBlurEffectStyle.ExtraLight))
-        view.addSubview(headerView);
-
-        headerView.snp_makeConstraints { make in
-            make.top.equalTo(self.view.snp_top)
-            make.height.equalTo(ToolbarHeight + StatusBarHeight)
+    private func wrapInEffect(view: UIView, position: String) {
+        let effect = UIVisualEffectView(effect: UIBlurEffect(style: UIBlurEffectStyle.ExtraLight))
+        self.view.addSubview(effect);
+
+        effect.snp_makeConstraints { make in
+            if position == "top" {
+                make.top.equalTo(self.view.snp_top)
+                make.height.equalTo(ToolbarHeight + StatusBarHeight)
+            } else {
+                make.bottom.equalTo(self.view.snp_bottom)
+                make.height.equalTo(ToolbarHeight)
+            }
             make.leading.trailing.equalTo(self.view)
         }
 
-        toolbar = BrowserToolbar()
-        toolbar.backgroundColor = UIColor.clearColor()
-        headerView.addSubview(toolbar)
-
-        toolbar.snp_makeConstraints { make in
-            make.top.equalTo(headerView.snp_top)
-            make.left.equalTo(headerView.snp_left)
-            make.bottom.equalTo(headerView.snp_bottom)
-            make.right.equalTo(headerView.snp_right)
+        view.backgroundColor = UIColor.clearColor()
+        effect.addSubview(view)
+
+        view.snp_makeConstraints { make in
+            make.top.equalTo(effect.snp_top)
+            make.left.equalTo(effect.snp_left)
+            make.bottom.equalTo(effect.snp_bottom)
+            make.right.equalTo(effect.snp_right)
```

Comment:
You can replace all four of these lines with <code>make.edges.equalTo(effect)</code> (plus the obligatory <code>return</code> since it's a one-liner).